flashflood/util.py

```diff
@@ -146,3 +146,10 @@ def upload_object(s3_client: typing.Any,
         return True
     else:
         return False
+
+def update_object_tagging(s3_client: typing.Any,
```

Comment:
I think that there's a 10 tag limit for objects, is this a concern?  https://docs.aws.amazon.com/AmazonS3/latest/dev/object-tagging.html

Reply:
Other restrictions like:
`A tag key can be up to 128 Unicode characters in length, and tag values can be up to 256 Unicode characters in length.`

Should we check for these in the code here or leave it to AWS?

Reply:
This is a convenience method to avoid dealing with strange s3 tagging data formats. For now, let's leave it to S3 to enforce tagging limits.